src/lib/create-config.ts

```diff
@@ -326,13 +329,20 @@ const parseCspEnvironmentVariables = (): ICspDomainConfig => {
     const scriptSrc = process.env.CSP_ALLOWED_SCRIPT?.split(',') || [];
     const imgSrc = process.env.CSP_ALLOWED_IMG?.split(',') || [];
     const connectSrc = process.env.CSP_ALLOWED_CONNECT?.split(',') || [];
+    const mediaSrc = process.env.CSP_ALLOWED_MEDIA?.split(',') || [];
+    const objectSrc = process.env.CSP_ALLOWED_OBJECT?.split(',') || [];
+    const frameSrc = process.env.CSP_ALLOWED_FRAME?.split(',') || [];
```

Comment:
Up until now this was derived from "default CSP". If someone depended on default CSP to embed frames/objects/media this has a potential to break things.

Reply:
It makes sense to support all, but I really struggle to see the value of custom values here.